class-udm-updater.php

```diff
@@ -81,7 +85,19 @@ public function __construct($mothership, $muid, $relative_plugin_file, $options
 		add_action('load-plugins.php', array($this, 'load_plugins_php'));
 		add_action('core_upgrade_preamble', array($this, 'core_upgrade_preamble'));
 		
-		add_filter('auto_update_plugin', array($this, 'auto_update_plugin'), 10, 2);
+		/**
+		 * Maintain compatibility on all versions between WordPress and UDM, specifically since WordPress 5.5 and UDM 1.8.8
+		 * Due to the new WP's auto-updates interface in WordPress version 5.5, we need to maintain the auto update compatibility on all versions of WordPress and UDM
+		 */
+		$udm_options = $this->get_option($this->option_name);
+		if (empty($udm_options['run_replace_auto_update_option_once'])) {
+			$this->replace_auto_update_option();
+		}
+
+		if (version_compare($wp_version, '5.5', '<')) {
+			// Auto update plugin
```

Comment:
@andtown I think this comment is unhelpful, because the filter itself doesn't perform any updates - please remove it.